spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/jsbridge/SCBridge.java

```diff
@@ -0,0 +1,404 @@
+package com.boundlessgeo.spatialconnect.jsbridge;
+
+import android.location.Location;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import com.boundlessgeo.spatialconnect.geometries.SCBoundingBox;
+import com.boundlessgeo.spatialconnect.geometries.SCGeometry;
+import com.boundlessgeo.spatialconnect.geometries.SCGeometryFactory;
+import com.boundlessgeo.spatialconnect.geometries.SCSpatialFeature;
+import com.boundlessgeo.spatialconnect.query.SCGeometryPredicateComparison;
+import com.boundlessgeo.spatialconnect.query.SCPredicate;
+import com.boundlessgeo.spatialconnect.query.SCQueryFilter;
+import com.boundlessgeo.spatialconnect.services.SCSensorService;
+import com.boundlessgeo.spatialconnect.services.SCServiceManager;
+import com.boundlessgeo.spatialconnect.stores.SCDataStore;
+import com.boundlessgeo.spatialconnect.stores.SCKeyTuple;
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.bridge.WritableArray;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+
+import java.io.UnsupportedEncodingException;
+import java.util.List;
+
+import rx.Subscriber;
+import rx.functions.Action1;
+import rx.schedulers.Schedulers;
+
+/**
+ * This module handles messages sent from Javascript.
+ */
+public class SCBridge extends ReactContextBaseJavaModule {
+
+    private final String LOG_TAG = SCBridge.class.getSimpleName();
+    private SCServiceManager manager;
+    private ReactContext reactContext;
+
+    public SCBridge(ReactApplicationContext reactContext) {
+        super(reactContext);
+        // TODO: this should be a singleton with a getInstance() method or injected with dagger
+        this.manager = new SCServiceManager(reactContext.getApplicationContext());
+        this.manager.startAllServices();
```

Comment:
We should expose this method through the bridge instead of starting all services by default